db/redis_test.go

```diff
@@ -0,0 +1,166 @@
+package db
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis/v2"
+	"github.com/vulsio/go-kev/models"
+	"github.com/vulsio/go-kev/utils"
+	"golang.org/x/xerrors"
+)
+
+func setupRedis() (*miniredis.Miniredis, DB, error) {
+	s, err := miniredis.Run()
+	if err != nil {
+		return nil, nil, xerrors.Errorf("Failed to run miniredis: %w", err)
+	}
+	driver, err := NewDB("redis", "redis://"+s.Addr(), false, Option{})
+	if err != nil {
+		return nil, nil, xerrors.Errorf("Failed to new db: %w", err)
+	}
+	return s, driver, nil
+}
+
+func teardownRedis(s *miniredis.Miniredis, driver DB) {
+	s.Close()
+	_ = driver.CloseDB()
+}
+
+func TestGetKEVByMultiCveID(t *testing.T) {
+	t.Parallel()
+	s, driver, err := setupRedis()
+	if err != nil {
+		t.Errorf("Failed to prepare redis: %s", err)
```

Comment:
If redis cannot be set up, testing cannot continue, so Fatalf seems more suitable than Errorf.
```suggestion
		t.Fatalf("Failed to prepare redis: %s", err)
```